CONTRIBUTING.md

```diff
@@ -10,33 +10,73 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 ```sh
 $ git clone https://github.com/netlify/netlify-cms
 $ cd netlify-cms
-$ yarn
+$ yarn start
 ```
 
-## Building
+## Available scripts
+
+### `bootstrap`
+
+Installs `yarn` dependencies and bootstraps the `lerna` packages.
+
+#### Usage
 
 ```sh
-$ npm run build
+$ yarn bootstrap
 ```
 
-## Testing
+### `watch`
+
+Watches all `lerna` packages and transpiles them on change.
```

Comment:
Same for the rest of the lines -- let's just say "CMS packages", or just "packages".